controllers/suite_test.go

```diff
@@ -824,7 +824,7 @@ func setup() error {
 	if result.Response.StatusCode != 204 {
 		_, err = resourceGroupManager.CreateGroup(context.Background(), resourceGroupName, resourcegroupLocation)
 		if err != nil {
-			return fmt.Errorf("ResourceGroup creation failed")
+			return fmt.Errorf("ResourceGroup creation failed: %v", err)
```

Comment:
Suggest using %w to create a nested error object (preserves `err` in case it contains any extended information)